lib/system.ml

```diff
@@ -312,3 +300,8 @@ let with_time ~batch f x =
     let msg2 = if batch then "" else " (failure)" in
     Feedback.msg_info (str msg ++ fmt_time_difference tstart tend ++ str msg2);
     raise e
+
+let get_toplevel_path top =
+  let dir = Filename.dirname Sys.argv.(0) in
+  let eff = if Dynlink.is_native then ".opt" else ".byte" in
```

Comment:
This is non necessary once the makefile is fixed as I suggest

Reply:
Umm, I think this is dangerous, I am OK with shipping a "default" copy even if that is useless and has dangerous implications [`coqtop.byte` and `coqtop.opt` don't behave the same in a few cases so users should be aware].

But IMHO it is very important to ensure that `coqide.byte` is actually starting `idetop.byte`, otherwise you are gonna be in trouble soon wrt testing.